llm/run.sub

```diff
@@ -0,0 +1,37 @@
+JobBatchName            = "LLM training template"
+
+universe                = docker
+docker_image            = ghcr.io/jasonlo/chtc_condor:latest
```

Comment:
Will there be any harm to your GitHub account if CHTC users pull your container and do not create their own? I'm not very familiar with their rate and bandwidth limits.